build/manifest/main.go

```diff
@@ -10,18 +10,22 @@ import (
 	"github.com/pkg/errors"
 )
 
-const pluginIdGoFileTemplate = `package main
+const pluginIDGoFileTemplate = `// This file is automatically generated. Do not modify it manually.
```

Comment:
The text should be "Code generated by <>. DO NOT EDIT." As per: https://golang.org/s/generatedcode.

Reply:
See above: https://github.com/mattermost/mattermost-plugin-github/pull/252#discussion_r418470854